src/store/index.ts

```diff
@@ -2,20 +2,18 @@ import { configureStore, ThunkAction, Action } from "@reduxjs/toolkit";
 import userReducer from "./reducer/userSlice";
 import tileReducer from "./reducer/tileSlice";
 import courtColorReducer from "./reducer/courtColorSlice";
-import rulerControlReducer from "./reducer/rulerControlSlice";
+import designPageButtonReducer from "./reducer/designPageButtonSlice";
```

Comment:
The name of this file can be more generic or specific. The logic in this slice is all boolean which means toggles, and not all of these booleans are on design page( there is one shopping cart toggle)